### DISCUSSION TITLE
Daily note template issue

### DISCUSSION THREAD

**Discussion by grannymaster** [TriliumNext/Trilium#4808] [Q&A] [answered]
Hi, 
I want to create a template which would be applied to each daily note. I've read this section many times, but for the life of me can't figure it out https://github.com/zadam/trilium/wiki/Day-notes#templates

I have separate file that I would use a template for daily notes like this:
![image](https://github.com/zadam/trilium/assets/79359055/0f0739ae-60e9-4fee-bf9c-6970efe8b756)

Can someone explain to me like I'm 5, what attributes I need to put where. I can't figure it out.

**Comment by zerebos** [comment, marked as answer]
On your calendar note (called `Calendar` in your example) you should see a `#calendarRoot` attribute, you need to add a new `~dateTemplate` attribute that points to your template. You can do this through the UI by adding a relation.

**Reply by grannymaster** [reply]
Ah I see, thank you! Got it working!

**Comment by grannymaster** [comment]
Got it working, thanks to @rauenzi
